ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used: python3.6+
+* Environment name and version (e.g. PHP 5.4 on nginx 1.9.1):
```

Comment:
Let's take out the PHP example as that is potentially confusing. Ideally they are using anaconda so write something like "Python 3.7.1 on conda 4.6.3"

Reply:
> Let's take out the PHP example as that is potentially confusing. Ideally they are using anaconda so write something like "Python 3.7.1 on conda 4.6.3"
Could you check my fix? I have just changed "PHP 5.4 on nginx 1.9.1" to "Python 3.7.1 on conda 4.6.3"